lib/settings_base.py

```diff
@@ -80,6 +80,9 @@ def path(*folders):
 DATABASES['default']['TEST_CHARSET'] = 'utf8'
 DATABASES['default']['TEST_COLLATION'] = 'utf8_general_ci'
 
+# Run all views in a transaction unless they are decorated not to.
+ATOMIC_REQUESTS = True
```

Comment:
@mstriemer the `change_slug()` scenario you illustrated is unlikely because any type of destructive request (post, patch, delete, etc) will pin the user [to master DB automatically](https://github.com/jbalogh/django-multidb-router/blob/master/multidb/middleware.py#L34).